components/service/glean/src/main/java/mozilla/components/service/glean/utils/DateUtils.kt

```diff
@@ -44,7 +44,7 @@ internal fun getISOTimeString(
 /**
  * Generate an ISO8601 compliant time string for the given time.
  *
- * @param date the [Date] object to convert to string
+ * @param calendar the [Date] object to convert to string
```

Comment:
```suggestion
 * @param calendar the [Calendar] object to convert to string
```

Reply:
Thanks for catching this out-of-sync comment!

Reply:
My pleasure ;) I'll add this to my next push.